app/models/user.rb

```diff
@@ -316,9 +316,17 @@ def liked_pages
   end
 
   def send_digest_email
-    top_picks = content_followed_in_period(1.week.ago, Time.now)
-    if top_picks.count.positive?
-      SubscriptionMailer.send_digest(id, top_picks).deliver_now
+    top_picks = []
+    freq = 1
+    if self.has_tag('digest:daily')
+      top_picks = content_followed_in_period(1.day.ago)
+      freq = 0
+    elsif self.has_tag('digest:weekly')
+      top_picks = content_followed_in_period(Time.now - 1.week, Time.now)
+      freq = 1
+    end
+    if !top_picks.nil? && top_picks.count > 0
+      SubscriptionMailer.send_digest(id, top_picks, freq).deliver_now
```

Comment:
aha, so could it be that the test was set up such that the test user doesn't actually have these tags, and so is now not getting the digest email? 

Also, if you wouldn't mind changing `top_picks` to `nodes` or `subscribed_content` perhaps, that'd be nice, thanks!

Reply:
Oh it could be that! Yeah sure, I will change it here as well, as I did in the mailer!